add-repos.sh

```diff
@@ -73,35 +73,11 @@ do
     mkdir -p "$clone_dir"
     cd "$clone_dir" || exit
 
-    if ! git clone --depth 1 --no-checkout "https://github.com/$REPO.git"; then
+    if ! git clone --depth 1 --no-checkout "https://$SCM/$REPO.git"; then
```

Comment:
I can see this was here before already, but I think we give enterprise the option of clone via http(s) or ssh; do we want to keep this as it is here?

Reply:
Yup, good point. why not :)